Cargo.lock

```diff
@@ -54,9 +54,9 @@ checksum = "d468802bab17cbc0cc575e9b053f41e72aa36bfa6b7f55e3529ffa43161b97fa"
 
 [[package]]
 name = "backtrace"
-version = "0.3.65"
+version = "0.3.66"
```

Comment:
Could you please not updated unrelated packages?

Why does libc need updating, anyway?

Reply:
Ok, this was just `cargo update`. https://github.com/rust-lang/libc/pull/2862 was only included recently and I thought we'd need it for a passthrough.

Reply:
No that won't work. libc calls the external fn, so if Miri implements the external fn by calling libc, at best we'll get an infinite loop.

Reply:
Also Miri this lock file is only relevant for things Miri needs to call *on the host*, and we certainly don't want to call the host libc for these functions.